hack/app-sre/build-push.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Build and push the operator image for direct deploy (no OLM).
+
+set -exv
+
+CURRENT_DIR=$(dirname $0)
+
+BASE_IMG="hive"
+# This must correspond to the REGISTRY_IMG variable defined in the
+# app-interface saas file: data/services/hive/cicd/ci-int/saas-hive.yaml
+QUAY_IMAGE="quay.io/app-sre/${BASE_IMG}"
+IMG="${BASE_IMG}:latest"
+
+# This must correspond to the IMAGE_TAG generated by app-sre: As long as we
+# *don't* set `use_channel_in_image_tag: true` in the saas file, this is the
+# first $NUM digits of the current commit hash. $NUM is 7 by default (if we
+# don't override it by setting `hash_length` in the saas file).
+GIT_HASH=`git rev-parse --short=7 HEAD`
+
```

Comment:
If we're going to ref images by digest, we need to [short out if the image exists](https://github.com/openshift/boilerplate/blob/328a6a5330ca758439829122dbdfafca9dcc79cf/boilerplate/_lib/common.sh#L85), both here and in app_sre_build_deploy.sh.

Reply:
n/a, but left TODO